spec/sum.js

```diff
@@ -1,4 +1,4 @@
-function sum(a, b) {
-  return a + b;
+function sum(x, y) {
```

Comment:
@ctong5 
@esme 

I love to see this small tests to ensure that Jest is working, and that you weren't starting too big. Seriously, this makes me so proud!

In general, I would avoid committing files that you've brought into your repo just to locally test whether some functionality is working. If you plan to leave them in, I'd name them with the same extension that you keep in your .gitignore file so that these files unrelated to the repo (and related to learning a framework) are not mistakenly committed.